Taskfile.yml

```diff
@@ -205,6 +205,8 @@ tasks:
     #   - "**/*.lock"
     #   - "**/*.snap"
     cmds:
+      # Build prqlc first for assert_cmd tests
+      - cargo build -p prqlc
```

Comment:
I need to build prqlc before testing also in the GitHub Action workflow to use `assert_cmd`...